.vuepress/config.js

```diff
@@ -100,6 +99,52 @@ module.exports = {
             },
           ],
         },
+        {
+          title: '📕 Cookbooks',
+          path: '/learn/cookbooks/',
+          collapsable: false,
+          children: [
+            {
+              title: 'Run in production',
+              path: '/learn/cookbooks/running_production',
```

Comment:
Don't know if it's just me, but I feel like this article has too many headers to work w/ the sidebar (plus, one of them is called "Content of this article", which just looks really strange out of context 🤓).

```suggestion
              title: 'Run in production',
              path: '/learn/cookbooks/running_production',
              sidebarDepth: 0,
```